dashboard/db/migrate/20200403195509_create_lesson_groups.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLessonGroups < ActiveRecord::Migration[5.0]
+  def change
+    create_table :lesson_groups do |t|
+      t.string :name
```

Comment:
we should think about how we want to sync these across environments. probably we will want to use a GUID, so that we do not have the same problems today that we have with renaming stages. What I would suggest is, that it is ok to use name for now so you can get things working. but then once you go to implement i18n, you should replace this with a GUID, which is then used as the key to look up translations in the .yml file. the doc linked from this work item regarding stage renaming may have some useful background: [PLAT-64](https://codedotorg.atlassian.net/browse/PLAT-64)

Reply:
just to add a bit more... scenarios to consider:

1. if we were to use the name as the cross-environment key for translations, and then we changed the name, we might break translations which were based on the old name. 
2. once we start allowing sections to hide lesson groups, we wouldn't want the lesson group to reappear if the curriculum writer changed the lesson group's name

happy to discuss more as this work item gets closer.

Reply:
sorry to brain dump here... I want to keep this thread in once place, but you don't need to worry about it in this PR.

It's also worth considering not doing this, in case we do not really care about the above scenarios, and want to keep the DSL simpler. it would work like this:

first, the levelbuilder would write:
```
stage 'Lesson 1', lesson_group: 'foo'
```
after saving, it would have a GUID added to it:
```
stage 'Lesson 1', lesson_group: 'foo', lesson_group_guid: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
```
if they keep the GUID but change the lesson_group name, they keep the translations and it stays hidden for any sections that hid it. 

we will most likely need GUIDs for stages, after which we might have lines looking like this:
```
stage 'Lesson 1', guid: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX', lesson_group: 'foo', lesson_group_guid: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
```

so if we don't care about the lesson group name-changing scenarios, then it'd be nice to leave them out and only have this:
```
stage 'Lesson 1', guid: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX', lesson_group: 'foo'
```

Reply:
I wrote this in the PLAT-64 JIRA also, but: I think there'd be a lot of value in having human readable unique IDs instead of GUIDs.